presto-docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -2,7 +2,7 @@
 Hive Connector with Azure Storage
 =================================
 
-The :doc:`hive` can be configured to query
+The :doc:`hive` connector can be configured to query
```

Comment:
This link adds Hive Connector automatically... 

If you want to control the text you can use

```
:doc:`Hive connector <hive>`
```

Reply:
Oops; fixed.